cmd/srcd/cmd/sql_test.go

```diff
@@ -1,28 +1,218 @@
-// +build integration
```

Comment:
is this intentional? if so, why?

Reply:
F**k, no, not intentional. Thanks.

Reply:
Fixed [here](https://github.com/src-d/engine/pull/398/commits/6a16536801a877db39deb7447e2dea550540f83c).